templates/cdk.master.auth-webhook.py

```diff
@@ -3,14 +3,20 @@
 import csv
 import json
 import logging
-import requests
+import aiohttp
 from base64 import b64decode
 from copy import deepcopy
-from flask import Flask, request, jsonify
 from pathlib import Path
 from subprocess import check_call, check_output, CalledProcessError, TimeoutExpired
 from yaml import safe_load
-app = Flask(__name__)
+
+
+AWS_IAM_ENDPOINT = '{{ aws_iam_endpoint }}'
+KEYSTONE_ENDPOINT = '{{ keystone_endpoint }}'
+CUSTOM_AUTHN_ENDPOINT = '{{ custom_authn_endpoint }}'
```

Comment:
```suggestion
AWS_IAM_ENDPOINT = '{{ aws_iam_endpoint if aws_iam_endpoint }}'
KEYSTONE_ENDPOINT = '{{ keystone_endpoint if keystone_endpoint }}'
CUSTOM_AUTHN_ENDPOINT = '{{ custom_authn_endpoint if custom_authn_endpoint }}'
```

Pretty sure you'll get a bunch of `'None'` strings because k8s-master is setting these context keys to `None`.

```python
>>> Template("{{ var }}").render(var=None)
'None'
>>> Template("{{ var if var }}").render(var=None)
''
```
This happened in the previous impl too, but the `foo_endpoint` handlers never got called because they were guarded with a `{% if foo_endpoint %}`.